src/commands/targeting/update.ts

```diff
@@ -0,0 +1,120 @@
+import { Args, Flags } from '@oclif/core'
+import UpdateCommand from '../updateCommand'
+import { EnvironmentPromptResult, FeaturePromptResult, environmentPrompt, featurePrompt } from '../../ui/prompts'
+import inquirer from '../../ui/autocomplete'
+import {
+    UpdateTargetingParamsInput,
+    fetchTargetingForFeature,
+    updateFeatureConfigForEnvironment
+} from '../../api/targeting'
+import { TargetingListOptions } from '../../ui/prompts/listPrompts/targetingListPrompt'
+import { FeatureConfig, UpdateTargetParams } from '../../api/schemas'
+import Base from '../base'
+import { validateSync } from 'class-validator'
+import { plainToClass } from 'class-transformer'
+import { reportValidationErrors } from '../../utils/reportValidationErrors'
+
+export default class UpdateTargeting extends Base {
+    static hidden = false
+    static description = 'Update Targeting rules'
+
+    static args = {
+        feature: Args.string({
+            description: 'The Feature for the Targeting Rule.'
+        }),
+        environment: Args.string({
+            description: 'The Environment where the Targeting Rule will be updated.'
+        }),
+    }
+
+    static flags = {
+        ...UpdateCommand.flags,
+        targets: Flags.string({
+            description: 'List of trageting rules'
+        }),
+    }
+
+    public async run(): Promise<void> {
+        const { args, flags } = await this.parse(UpdateTargeting)
+        const { feature, environment } = args
+        const { targets, headless } = flags
+
+        await this.requireProject()
+
+        if (headless && (!feature || !environment)) {
+            this.writer.showError('The key, feature and environment arguments are required')
```

Comment:
doesn't look like a key is required here?
```suggestion
            this.writer.showError('The feature and environment arguments are required')
```

Reply:
looks like this still needs to be updated